Tasks/AzureKeyVault/models/KeyVaultTaskParameters.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference path="../../../definitions/node.d.ts" /> 
```

Comment:
each task needs it's own typings.  see shellscript task.  this causes updates needing all teams to revalidate all tasks.